docs/UserGuide.md

```diff
@@ -911,6 +911,10 @@ To circumvent this, you can add additional details to the name to differentiate
 ## Known issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
+2. **In calendar window**:
+   * the default size only show a maximum of 2 clients per day
```

Comment:
I feel it would be better to number these then line 917 shouldnt be a bullet point

for example:
* 1) The default size of the window only shows a maximum of 2 clients per day
* 2) The default size of the window does not show time of the appointment
This can be fixed by manually resizing the calendar window to display more information

Reply:
hmm is the new format better?

Reply:
better now! just need to capitalise the "the" at the start of both points (i.e from "the default size..." to "The default size...")
and it should be "shows", not "show"" (for the first one) (i.e from "... only show..." to "... only shows... "

Reply:
done!

Reply:
Alright! Merging after checks